.devcontainer/Dockerfile

```diff
@@ -42,6 +42,18 @@ RUN useradd -s /bin/bash -u $USER_UID -g $USER_GID -G docker -m $USERNAME
 RUN echo $USERNAME ALL=\(root\) NOPASSWD:ALL > /etc/sudoers.d/$USERNAME
 RUN chmod 0440 /etc/sudoers.d/$USERNAME
 
+# Python 3.9 and PIP
```

Comment:
Does this mean it only work with python 3.9

Reply:
On macOS it uses the Pigweed environment, but for Linux it needs 3.9, because I was unable to get the Pigweed environment to install the same version as macOS successfully. I will need help from someone at google to achieve this. I also want to update the macOS env to python3.9.